ui/de-lib/src/main/java/org/iplantc/de/client/models/sharing/Sharing.java

```diff
@@ -9,29 +14,67 @@
  */
 public class Sharing {
```

Comment:
Not sure if we want to use Javascript Overlay type here. Sharing is just POJO that i am using store and retrieve information throughout the sharing logic That's never parsed into JSON. That being said I might need a new Model just for encode and decode the information that I send over wire.